src/main/java/org/terasology/polyworld/rp/WorldRegionFacetProvider.java

```diff
@@ -46,19 +46,22 @@
 
     private final ReentrantReadWriteLock lock = new ReentrantReadWriteLock(false);
 
+    public static final int SECTOR_SIZE = 1024;
+    public static final int SECTOR_POWER = Integer.numberOfTrailingZeros(SECTOR_SIZE);
```

Comment:
Where do these values come from?

Reply:
I didn't want the dependency to Sector so the sector area is defined her and allocated as a BlockRegion

https://github.com/Terasology/CommonWorld/blob/93cc9918d28b29d9c64e3a8d884edcbc82c6dccb/src/main/java/org/terasology/commonworld/Sector.java#L33